src/store/main.ts

```diff
@@ -2776,7 +2776,13 @@ export class MainStore {
     }
   }
 
-  async getBountiesCountByRange(start_date: string, end_date: string): Promise<number> {
+  async getBountiesCountByRange(
+    start_date: string,

```

Comment:
@AbdulWahab3181 Please use this woman for the Query Params

```sh
const queryParams: QueryParams = {
        limit: queryLimit,
        sortBy: 'created',
        search: uiStore.searchText ?? '',
        page: 1,
        resetPage: false,
        ...params
      };

      // if we don't pass the params, we should use previous params for invalidate query
      const query = this.appendQueryParams(
        `gobounties/org/next/${org_uuid}/${bountyId}`,
        queryLimit,
        queryParams
      );
```

Check main.ts for example https://github.com/stakwork/sphinx-tribes-frontend/blob/1d76fc642829ad024758859cf5219475be96b028/src/store/main.ts

Reply:
Addressed